algorithms/src/Kokkos_Random.hpp

```diff
@@ -2059,7 +2059,8 @@ void fill_random(ViewType a, RandomPool g,
                  typename ViewType::const_value_type range) {
   int64_t LDA = a.extent(0);
   if (LDA > 0)
-    parallel_for((LDA + 127) / 128,
+    parallel_for("kokkos.algorithms.fill_random",
+                 Kokkos::RangePolicy<>(0, (LDA + 127) / 128),
```

Comment:
Args this is actually not the same, since now the Policy and Functor compete for setting the execution space. You don't need to add RangePolicy here btw. Names work with integer as the second argument.

Reply:
So if I just leave it as 

```c++
parallel_for(name, 0, upper_bound, functor_or_lambda);
```

Then we're good?

Reply:
Or is that an alias for creating a defaulted RangePolicy (it doesn't look like it, I just want to double check)

Reply:
`parallel_for(name,upper_bound,functor_or_lambda);`